.github/workflows/constraints.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         uses: peter-evans/create-pull-request@v3
         if: ${{ success() }}
         with:
+          branch: update-obo-epm-${{ github.run_id }}
```

Comment:
```suggestion
          branch: update-constraints-${{ github.run_id }}
```

Reply:
Otherwise I don’t see the point of the PR! :D

Reply:
Actually the run_id was not even necessary - this way it does not update the existing PR, which it should, when running again.